tests/system/conftest.py

```diff
@@ -193,11 +207,15 @@ def shared_database(shared_instance, database_operation_timeout, database_dialec
         operation.result(database_operation_timeout)  # raises on failure / timeout.
 
     else:
+        ddl_statements = _helpers.DDL_STATEMENTS
+        if not _helpers.USE_EMULATOR:
+            ddl_statements = ddl_statements + _helpers.PROTO_COLUMNS_DDL_STATEMENTS
         database = shared_instance.database(
             database_name,
-            ddl_statements=_helpers.DDL_STATEMENTS,
+            ddl_statements=ddl_statements,
             pool=pool,
             database_dialect=database_dialect,
+            proto_descriptors=proto_descriptor_file,
```

Comment:
Emulator doesn't support this

Reply:
Emulator support for proto columns is in progress and should be available before mainline merge.